recruiter/views.py

```diff
@@ -70,5 +70,27 @@ def recruiterRegister(request):
     return render(request, 'recruiter/registerRecruiter.html', context)
 
 
+def companyRegister(request):
+    if request.method == 'POST':
+        form = CompanyRegistrationForm(request.POST)
+        if form.is_valid():
+            form.save()
+            return redirect('/company_created_successfully')
+    else:
+        form = CompanyRegistrationForm()
+
+    context = {'form': form}
+    jobboard.views.add_navbar_links_to_context(request, context)
+    return render(request, 'recruiter/registerCompany.html', context)
+
+
+def company_created_successfully(request):
+    context = {}
+    jobboard.views.add_navbar_links_to_context(request, context)
+    return render(request, 'recruiter/company_created_successfully.html', context)
+
+
 def recruiter_created_successfully(request):
-    return render(request, 'recruiter/recruiter_created_successfully.html')
```

Comment:
I already submitted this change on: 
https://github.com/redhat-beyond/SJMaster/pull/77
So you can leave it as is for now.

Reply:
You told me to upload this on this PR and now to tell me to put it down?

Reply:
I didn't tell you to put this in the PR.. I asked you if you want to do it on your PR so I can close mine but you didn't respond so I left mine open.. If you want to add it I can close my PR but you need to add it in all the places, not just here (look at the changes in my PR).

Reply:
I'm deleting it.